omnibus/config/projects/agent.rb

```diff
@@ -53,7 +53,7 @@
   end
 
   if debian?
-    runtime_recommended_dependency 'datadog-signing-keys (>= 1.1.0)'
+    runtime_recommended_dependency 'datadog-signing-keys (>= 1:1.1.0)'
```

Comment:
do we need to/is it better to include the revision in the `version` as well (the `-1` prefix, e.g. `1:1.1.0-1`)

Reply:
The `-1` is implied here. If we did a `-2` release, we could bump this requirement to `1:1.1.0-2` to depend on that explicitly (IOW while for epoch, the default is `0`, for revision the default is `1` and that is correct for us).

Reply:
ok thanks for the clarification